easybuild/tools/filetools.py

```diff
@@ -296,7 +408,7 @@ def find_easyconfigs(path, ignore_dirs=None):
                 continue
 
             spec = os.path.join(dirpath, f)
-            _log.debug("Found easyconfig %s" % spec)
+            _log.debug("Found easyconfig %s", spec)
```

Comment:
As mentioned in my other replies: let's fix our `log.error` then?

Using (our custom) `log.error` is just the way this is done in the EasyBuild codebase, I understand it's different in our other `vsc` Python packages, but let's stick to it.

I'd like to stay away from throwing stack traces into people's faces when they trigger something EasyBuild knows to be wrong.